### PR TITLE
Keyword Search Endpoint

### PR DESCRIPTION
**Pull request by salsabiilashifa11**

### CODE REVIEW

**Comment by reymyr** [comment]
Ini gapapa masih ada entity extraction (library transformers sama pytorch?

**Review by reymyr — COMMENTED** [review]
Ini gapapa masih ada entity extraction (library transformers sama pytorch?

**Comment by salsabiilashifa11** [comment]
> Ini gapapa masih ada entity extraction (library transformers sama pytorch)?

Oh iya, removed myr, thanks

File: api/__init__.py [hunk 1]
```diff
@@ -4,11 +4,15 @@
...
 from api.document.v1.document import document_router
+from api.entity.v1.entity import entity_router
+from api.search.v1.search import search_router
 
 router = APIRouter()
 router.include_router(user_v1_router, prefix="/api/v1/users", tags=["User"])
 router.include_router(document_router, prefix="/api/v1/documents", tags=["Document"])
 router.include_router(auth_router, prefix="/auth", tags=["Auth"])
 router.include_router(elastic_router, prefix="/elastic", tags=["Elastic"])
+router.include_router(entity_router, prefix="/api/v1/entities", tags=["Entity"])
```
Comment (reymyr) [inline api/__init__.py:15]:
```suggestion
```

File: api/__init__.py [hunk 2]
```diff
@@ -4,11 +4,15 @@
 from api.auth.auth import auth_router
 from api.elastic.v1.elastic import elastic_router
 from api.document.v1.document import document_router
+from api.entity.v1.entity import entity_router
```
Comment (reymyr) [inline api/__init__.py:7]:
```suggestion

```

**Review by reymyr — APPROVED** [review]

**Review by AndhikaRei — APPROVED** [review]

**Review by AndhikaRei — APPROVED** [review]

**Pull request merged**